src/int.cr

```diff
@@ -486,6 +486,51 @@ struct Int
     io << type
   end
 
+  # Prints this integer as a binary value in a human-readable format using
+  # multiples of 1024 as prefixes.
+  #
+  # Values with binary measurements such as computer storage (e.g. RAM size) are
+  # typically expressed using unit prefixes based on 1024 (instead of multiples
+  # of 1000 as per SI standard). This method by default uses an extended range
+  # of the JEDEC units (`K`, `M`, `G`, `T`, `P`, `E`, `Z`, `Y`) which equals to
+  # the prefixes of the SI system except for uppercase `K` and is based on
+  # powers of 1024. The IEC standard prefixes (`ki`, `Mi`, `Gi`, `Ti`, `Pi`,
+  # `Ei`, `Zi`, `Yi`) will be used if *standard* is `:IEC`.
+  #
+  # ```
+  # 1.humanize_bytes                          # => "1B"
```

Comment:
Why do some outputs have precision and some don't?
Seems like this should be `1.0B` or anything that is `humanize` that does not have a remainder should be `1B`

Reply:
You can't have a fraction of a byte, so `1.0 bytes` makes no sense. This is a special case, because number of bytes are non-negative integers.
In contrast, most other units are measurements expressed as real numbers. For example, `1.0 meters` is completely legit and semantically different from `1 meter` (reduced precision).